src/gpu/create_beta_gk.cu

```diff
@@ -36,7 +36,7 @@ __global__ void create_beta_gk_gpu_kernel
     int const* beta_desc__,
     gpu_complex_type<T> const* beta_gk_t,
     T const* gkvec,
-    T const* atom_pos,
+    double const* atom_pos,
```

Comment:
@manhin321 can you also check the G+k vector coordinates are also always in `double`?